test/integration/router_test.go

```diff
@@ -233,6 +233,12 @@ func createAndStartRouterContainer(dockerCli *dockerClient.Client, masterIp stri
 		},
 	}
 
+	err = pullIfNotPresent(dockerCli, getRouterImage())
```

Comment:
It was failing hard.  I added this in so it wouldn't.  If we're ok with the failure then this PR can be closed.